packages/expo-modules-autolinking/src/reactNativeConfig/__tests__/reactNativeConfig-test.ts

```diff
@@ -200,6 +201,68 @@ describe(findDependencyRootsAsync, () => {
     `);
   });
 
+  describe(resolveEdgeToEdgeDependencyRoot, () => {
+    beforeAll(() => {
```

Comment:
maybe this should be `beforeEach` so that the test cases are independent of one another?